code/modules/vending/restock_tracking/restock_tracker.dm

```diff
@@ -0,0 +1,38 @@
+
+/**
+ * The restock tracker computer keeps tabs on which vending machines on station are running low on stock.
+ * It can be used by cargo technicians to quickly find out which machines need to be restocked, and rewards crew
+ */
+
+/obj/machinery/computer/restock_tracker
+	name = "restock tracker"
+	desc = "A computer that tracks how well stocked the station's vending machines are."
+	circuit = /obj/item/circuitboard/computer/restock
+
+/obj/machinery/computer/restock_tracker/ui_interact(mob/user, datum/tgui/ui)
+	. = ..()
+	ui = SStgui.try_update_ui(user, src, ui)
+	if(!ui)
+		ui = new(user, src, "Restock", name)
+		ui.open()
+
+/obj/machinery/computer/restock_tracker/ui_data(mob/living/carbon/human/user)
```

Comment:
I just noticed this is entirely copy pasted between the app and the computer. Why have the computer anymore? Just use a mod PC pre-loaded with the app?

Reply:
I lean towards soul, but...

Yeah, you're right...